### PR TITLE
Fix log lines

### PR DESCRIPTION
**Pull request by fernandog**
Split from the other PR

### CODE REVIEW

**Comment by fernandog** [comment]
@ratoaq2 are you working on this?

File: sickbeard/show_queue.py [hunk 1]
```diff
@@ -649,82 +651,97 @@ def run(self):
...
+                       (id=self.show.indexerid, error_msg=ex(e)), logger.WARNING)
         except Exception as e:
-            logger.log(u"Error loading IMDb info: " + ex(e), logger.ERROR)
-            logger.log(traceback.format_exc(), logger.DEBUG)
+            logger.log(u'{id}: Error loading IMDb info: {error_msg}'.format
+                       (id=self.show.indexerid, error_msg=ex(e)), logger.WARNING)
 
         # have to save show before reading episodes from db
         try:
+            logger.log(u'{id}: Saving show new info to database'.format(id=self.show.indexerid))
```
Comment (medariox) [inline sickbeard/show_queue.py]: ```Saving **new show** info to database```

File: sickbeard/tv.py [hunk 2]
```diff
@@ -383,18 +384,17 @@ def get_episode(self, season=None, episode=None, filepath=None, no_create=False,
...
+                logger.log(u'{id}: Multiple entries for absolute number: {absolute} in show: {show} found '.format
+                           (id=self.indexerid, absolute=absolute_number, show=self.name), logger.ERROR)
 
                 return None
             else:
-                logger.log(
-                    'No entries for absolute number: ' + str(absolute_number) + ' in show: ' + self.name + ' found.',
-                    logger.DEBUG)
+                logger.log(u'{id}: No entries for absolute number {absolute_number} in show: {show}'.format
+                           (id=self.indexerid, absolute_number=absolute_number, show=self.name), logger.DEBUG)
```
Comment (medariox) [inline sickbeard/tv.py]: Maybe call it `absolute` for consistency.

File: sickbeard/tv.py [hunk 3]
```diff
@@ -2200,13 +2228,16 @@ def __str__(self):
...
+
+            logger.log(u'{id}: The show dir is missing, unable to create metadata'.format
+                       (id=self.show.indexerid), logger.WARNING)
             return
 
         self.__create_nfo()
         self.__create_thumbnail()
 
         if self.check_for_meta_files():
+            logger.log(u'{id}: Metadata changes to database'.format(id=self.show.indexerid))
```
Comment (medariox) [inline sickbeard/tv.py]: ```Saving metadata changes to database``` ?

**Comment by medariox** [comment]
Approved

**Pull request merged**